tests/MockData/json_response/v3_content_scripts.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data": {
```

Comment:
Good idea of providing the mock response data directly here. We also do this in the new REST Exporter and it works really well. Having a direct response also makes your code more stable, since you fake a real scenario.